### DISCUSSION TITLE
How do you pronounce "preact"? 🙂

### DISCUSSION THREAD

**Discussion by octogonz** [preactjs/preact#3151] [General]
Do you say...
1. *PREE act*
2. *PEE react*
3. something else?

**Comment by JoviDeCroock** [comment]
I personally say number 1 😅 would love to hear what other people say

**Reply by marvinhagemeister** [reply]
Yup, 1) ist the correct answer. See @developit 's talk at JSConfEU 2017 https://youtu.be/LY6y3HbDVmg?t=95 at 1:35 .

**Reply by octogonz** [reply]
Cool, thanks!

**Comment by lfgraphics** [comment]
I used to say P React 😅
but a moment ago I googled about its pronunciation and found it is pre/pree act
and saw this thread too, found interesting and just landed here.
